http/http_test.go

```diff
@@ -0,0 +1,60 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"slices"
+	"testing"
+)
+
+func TestGetJSON(t *testing.T) {
+	t.Run("ABC", func(t *testing.T) {
+		t.Parallel()
+
+		slice := []int{0, 1}
+		byteIn, err := json.Marshal(slice)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		// testserver
+		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.WriteHeader(http.StatusOK)
+			fmt.Fprintln(w, byteIn)
+		}))
```

Comment:
that works, but that also works:

```suggestion
		// testserver
		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
			w.WriteHeader(http.StatusOK)
			fmt.Fprintln(w, `[0,1]`)
		}))
```

Reply:
Thanks for confirming my suspicion that my code was a bit complex.
Changed.